### PR TITLE
[Docs] How to unfreeze the backbone network

### PR DESCRIPTION
**Pull request by Czm369**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

User want to unfreeze backbone after freezing the backbone resnet in the config.

## Modification

Write a hook function.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/tutorials/how_to.md [hunk 1]
```diff
@@ -112,3 +112,63 @@ data = dict(
...
+        self.unfreeze_epoch = unfreeze_epoch
+
+    def before_train_epoch(self, runner):
+        # Unfreeze the backbone network.
+        if runner.epoch == self.unfreeze_epoch:
+            model = runner.model
+            if is_module_wrapper(model):
+                model = model.module
+            backbone = model.backbone
+            if backbone.frozen_stages >= 0:
```
Comment (hhaAndroid) [inline docs/en/tutorials/how_to.md:158]: Only valid for resnet

File: docs/en/tutorials/how_to.md [hunk 2]
```diff
@@ -112,3 +112,63 @@ data = dict(
     train=train_dataset
     )
 ```
+
+## Unfreeze backbone network after freezing the backbone in the config
```
Comment (hhaAndroid) [inline docs/en/tutorials/how_to.md:116]: Currently, this is only valid for resnet, should all backends be considered?

**Comment by ZwwWayne** [comment]
Kindly ping the progress.

**Review by zytx121 — APPROVED** [review]
LGTM

**Pull request merged**